entrypoint.sh

```diff
@@ -56,6 +60,10 @@ main() {
     git config --global url."https://".insteadOf git://
     ## $GITHUB_SERVER_URL is set as a default environment variable in all workflows, default is https://github.com
     git config --global url."$GITHUB_SERVER_URL/".insteadOf "git@${GITHUB_HOSTNAME}":
+
+    echo "Setting default branch to #{SOURCE_BRANCH_NAME}"
+    git config --global init.defaultBranch $SOURCE_BRANCH_NAME
```

Comment:
Are you sure we need this? This config only affects the git init command not the git pull command. And we don't care what the branch of the intermediate git repo is since it's pushed to only $PAGES_BRANCH

Reply:
Without this I'm getting 
![image](https://user-images.githubusercontent.com/26892280/215964440-7c5bbb17-3759-43d9-a09a-9d6822fd7739.png)
in the execution of `entrypoint.sh` which isn't fatal, but it is additional noise when trying to change or debug the action. Note that this won't go away if you're using the default value of `master` either.

I don't know. I let you decide on if we should keep it in.

Reply:
Let's remove it, since we have to keep the master branch as default.